src/content/energy-consumption/index.md

```diff
@@ -6,36 +6,36 @@ lang: en
-| Global data centers |                 200                 |          78,000x           | [source](https://www.iea.org/commentaries/data-centres-and-energy-from-global-headlines-to-local-headaches)                                       |
-| Netflix             |                0.45                 |            175x            | [source](https://s22.q4cdn.com/959853165/files/doc_downloads/2020/02/0220_Netflix_EnvironmentalSocialGovernanceReport_FINAL.pdf)                  |
-| Netflix             |                 94                  |          36,000x           | [source](https://theshiftproject.org/en/article/unsustainable-use-online-video/)                                                                  |
-| PayPal              |                0.26                 |            100x            | [source](https://app.impaakt.com/analyses/paypal-consumed-264100-mwh-of-energy-in-2020-24-from-non-renewable-sources-27261)                       |
-| Gaming in USA       |                 34                  |          13,000x           | [source](https://www.researchgate.net/publication/336909520_Toward_Greener_Gaming_Estimating_National_Energy_Use_and_Energy_Efficiency_Potential) |
-| PoW Ethereum        |                 78                  |          30,000x           | [source](https://digiconomist.net/ethereum-energy-consumption)                                                                                    |
-| PoS Ethereum        |               0.0026                |             1x             | [source](https://carbon-ratings.com/eth-report-2022)                                                                                              |
+It is complicated to get accurate estimates for energy consumption, especially when what is being measured has a complex supply chain or deployment details that influence its efficiency. For example, the energy consumption for platforms such as Netflix and Youtube vary depending upon whether they only include the energy used to run the hardware infrastructure required to store and deliver content (_direct expenditure_) or whether they include the expenditure required to - for example - produce content, run corporate offices, advertise, etc (_indirect expenditure_). Indirect usage could also include the energy required to consume content on end-user devices such as TVs, computers and mobiles, which in turn depends on which devices are used - there is some discussion this on [Carbon Brief](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix). In the table above, the value reported for Netflix is their self-reported total (_direct_ + _indirect_). Youtube estimates their own _direct_ energy expenditure at around [12 TWh/yr](https://www.gstatic.com/gumdrop/sustainability/google-2020-environmental-report.pdf).
```

Comment:
Tried to simplify the wording a little here. LMK what you think.

```suggestion
It is complicated to get accurate estimates for energy consumption, especially when what is being measured has a complex supply chain or deployment details that influence its efficiency. Consider Netflix and YouTube as examples. The energy they use is divided into two categories: direct and indirect. Direct energy is necessary to maintain their systems and deliver content to users. Indirect energy encompasses aspects such as producing shows, operating the company, advertising, and even the energy consumed while watching on devices like TVs or phones. 

You can learn more about these differences at [Carbon Brief](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix). In the table above, the number for Netflix includes both direct and indirect energy. YouTube, on the other hand, only provides us a number for their direct energy use, which is around [12 TWh/yr](https://www.gstatic.com/gumdrop/sustainability/google-2020-environmental-report.pdf).
```

Reply:
let me come back to this tomorrow, i think I've lost a bit of the nuance. Netflix's number is direct + indirect but does not include end users..

Reply:
```suggestion
It is complicated to get accurate estimates for energy consumption, especially when what is being measured has a complex supply chain or deployment details that influence its efficiency. consider Netflix or Youtube as examples. Estimates of their energy consumption vary depending upon  whether they only include the energy used to maintain their systems and deliver content to users (_direct expenditure_) or whether they include the expenditure required to produce content, run corporate offices, advertise, etc (_indirect expenditure_). Indirect usage could also include the energy required to consume content on end-user devices such as TVs, computers and mobiles, which in turn depends on which devices are used.

There is some discussion of this issue on [Carbon Brief](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix). In the table above, the value reported for Netflix includes their self-reported _direct_ and _indirect_ usage. Youtube only provides an estimate of their own _direct_ energy expenditure, which is around [12 TWh/yr](https://www.gstatic.com/gumdrop/sustainability/google-2020-environmental-report.pdf).
```